### PR TITLE
Replace model dependent evaluation result with report file since that contains all the evaluation information

### PR DESCRIPTION
**Pull request by bauersimon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,12 +131,7 @@ Total coverage 100.000000%
 
 </details>
 
-The execution by default also creates an evaluation file `evaluation.csv` that contains:
-
-```
-model,files-total,files-executed,files-problems,coverage-statement
-openrouter/anthropic/claude-3-opus,1,1,0,100
-```
+The execution by default also creates an report file `REPORT.md` that contains additional evaluation results.
```
Comment (zimmski) [inline README.md:134]: `evaluation results.` -> `evaluation results and links to individual result files.`

To make it clear that there are other files

File: README.md [hunk 2]
```diff
@@ -148,10 +143,10 @@ To add new tasks to the benchmark, add features, or fix bugs, you'll need a deve
 
 First of all, thank you for thinking about contributing! There are multiple ways to contribute:
 
--   Add more files to existing language repositories.
```
Comment (zimmski) [inline README.md:151]: Please use prettier to format.

Comment (bauersimon) [reply README.md:151]:
```
prettier -w README.md
README.md 80ms (unchanged)
```

:thinking:

Comment (bauersimon) [reply README.md:151]: Will just revert this then...

**Pull request merged**